docs/DafnyRef/UserGuide.md

```diff
@@ -626,31 +1030,71 @@ TO BE WRITTEN
    * `py` - Python
```

Comment:
This is rendered differently because of the missing empty lines. Not sure, whether it should change.